src/Modules/OrchardCore.Commerce/Services/StripePaymentService.cs

```diff
@@ -333,6 +331,54 @@ public async Task<IEnumerable<OrderLineItem>> CreateOrderLineItemsAsync(Shopping
         return lineItems;
     }
 
+    private static void AlterOrder(
+        ContentItem order,
+        PaymentIntent paymentIntent,
+        Amount defaultTotal,
+        ShoppingCartViewModel cartViewModel,
+        IEnumerable<OrderLineItem> lineItems)
+    {
+        order.Alter<OrderPart>(orderPart =>
+        {
+            orderPart.Charges.Clear();
```

Comment:
Can Charges be non-empty at this point?

Reply:
I think it's possible when an Order was paid for once and then set to Pending again.

Reply:
Sounds like fraud but that's none of my business...